ReactSkia/RSkThirdPartyNativeModuleProvider.cpp

```diff
@@ -12,19 +12,22 @@ namespace react {
 RSkLegacyNativeModuleProviderProtocol RSkThirdparyNativeModuleCreatorClassWithName(std::string moduleName) {
   // This function and hash table must be generated by codegen with all the linked thirdparty Native modules
   static std::unordered_map<std::string, RSkLegacyNativeModuleProviderProtocol> NativeModulesClassMap = {
+	{std::string("RNKeyEvent"), RNKeyEventModuleCls},
```

Comment:
my understanding is these entries will be added by codegen or any other in future. So we do not add these thirdparty package entries